File: solutions/uri/1008/1008.clj
(ns main)

(defn main []
  (let [a (Integer/parseInt (read-line))
        b (Integer/parseInt (read-line))
        c (Double/parseDouble (read-line))]
    (println "NUMBER =" a)
    (println "SALARY = U$" (format "%.2f" (* c b)))))

(main)

File: solutions/uri/1008/1008.c
#include <stdint.h>
#include <stdio.h>

int main() {
    int16_t a, b;
    float c;

    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%f", &c);

    printf("NUMBER = %d\n", a);
    printf("SALARY = U$ %.2f\n", c * b);

    return 0;
}
